test/Header/HeaderValueTest.php

```diff
@@ -56,7 +57,11 @@ public function validateValues()
             ["This is a\r\r test", 'assertFalse'],
             ["This is a \r\r\n test", 'assertFalse'],
             ["This is a \r\n\r\ntest", 'assertFalse'],
-            ["This is a \r\n\n\r\n test", 'assertFalse']
+            ["This is a \r\n\n\r\n test", 'assertFalse'],
+            ["This\tis\ta test", 'assertTrue'],
+            ["This is\ta \r\n test", 'assertTrue'],
+            ["This\tis\ta\ntest", 'assertFalse'],
+            ["This is a \r\t\n \r\n test", 'assertFalse'],
```

Comment:
I couldn't make it assert true. Breaks other the tests. Can you please take a look. Much appreciated.